docs/store/data-items.md

```diff
@@ -32,7 +32,7 @@ prep_data_run = data_prep_func.run(name='prep_data',
                                    params={'label_column': 'label'})
 ```
 
-In order to call the function with an `input` you can use the `inputs` dictionary attribute. In order to pass
+To call the function with an `input` you can use the `inputs` dictionary attribute. To pass
 a simple parameter, use the `params` dictionary attribute. The input value is the specific item uri
 (per data store schema) as explained in [Shared data stores](../store/datastore.html#shared-data-stores).
```

Comment:
You should mention here that since 1.3 `DataItem`s will be automatically parsed to the hinted type when a type hint is available (like I wrote above)

Reply:
updated